libs/transactions/src/lib/components/text-filter/text-filter.component.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { TextFilterComponent } from './text-filter.component';
```

Comment:
I miss some test about what this components does filter...

Reply:
I can add DOM manipulation testing if needed. Angular Test Bed with Karma/Jasmine may be better for this approach.

I made the change because PlayWright would test the the interaction of filtering and there is no functionality in this TypeScript file to test.

Reply:
How will it test this filtering.

Reply:
The filter pipe will test the logic of the filtering. During an e2e test using Playwright, the filtering of the DOM would be tested. Jest without Angular Test bed focuses purely on the logic inside of each individual component which greatly improves speed of running Unit Tests.